### DISCUSSION TITLE
Set a default profile / Track last logged in profile

### DISCUSSION THREAD

**Discussion by antonionardella** [iotaledger/firefly#1562] [Features / Enhancements]
## Description

An option to set the default profile for the wallet.

## Motivation

- Should the user have only one profile in his wallet, there should be an option to set a default profile, to directly ask for the PIN, instead of having an additional click to select _the only available profile_
- Should I have one _daily_ account and a _savings_ account, I might want to open the daily account directly and log out to switch to the savings account, only when needed.

## Requirements

Write a list of what you want this feature to do.

1. Settings under profile to select the default profile
2. When default profile is set, Firefly should ask straight for the PIN when it is opened

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

No.

**Comment by rafaeldjpbrochado** [comment]
Agreed. That and suggestion https://github.com/iotaledger/firefly/issues/332 will both significantly reduce the time it takes to get into the app.

**Comment by rafaeldjpbrochado** [comment]
In fact, here's how I think I would do it:

- When you open up the app, it **always** defaults back to the last profile you used. However, right there on the landing page there would be an option to change profile (as well as an option titled "Forgot your PIN?" which would let users sign in using their password or mnemonic seed).

![fireflyloginscreen](https://user-images.githubusercontent.com/39466455/111057752-d7a70300-8457-11eb-902e-af425c6c3ead.png)

**Comment by maxwellmattryan** [comment]
Hey again @antonionardella, also moved this over to a [discussion](https://github.com/iotaledger/firefly/discussions/1482) 👍
